ui/components-react/src/test/filter-builder/FilterBuilderRuleGroup.test.tsx

```diff
@@ -248,4 +249,22 @@ describe("PropertyFilterBuilderRuleGroupRenderer", () => {
       PropertyFilterRuleGroupOperator.Or
     );
   });
+
+  it("focuses rule property after adding new rule", async () => {
+    const actions = new PropertyFilterBuilderActions(sinon.spy());
+    const { getByTestId, container } = renderWithContext(
+      <PropertyFilterBuilderRuleGroupRenderer {...defaultProps} />,
+      { actions }
+    );
+
+    const addRuleButton = getByTestId("rule-group-add-rule");
+    addRuleButton.click();
+
+    await waitFor(
+      () =>
+        expect(
```

Comment:
I would suggest checking this with `expect(container.querySelector(".rule-property input") === container.ownerDocument.activeElement).to.be.true;`

Reply:
fixed